task/control-plane-agent/src/inventory/our_inventory.rs

```diff
@@ -0,0 +1,111 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! Submodule that defines logical components we claim to have (as opposed to
+//! components entirely managed by other tasks, like I2C devices).
+
+use gateway_messages::{
+    sp_impl::DeviceDescription, DeviceCapabilities, DevicePresence, SpComponent,
+};
+
+pub(super) struct OurInventory {}
```

Comment:
If we're making `DEVICES` `pub(super)` to consolidate the compile-time length checks, then this `struct` seems superfluous versus storing `DEVICES` directly in the `struct Inventory`.

```rust
ours: OurInventory,
```
becomes
```rust
ours: &'static [DeviceDescriptor], // do we need `<'static>` here?
```